src/components/LandingPage/PillarInfoBanner/PillarInfoBanner.component.tsx

```diff
@@ -0,0 +1,101 @@
+import influencePillarImage from "@/assets/images/pillars-info/wplyw.png";
+import relationsPillarImage from "@/assets/images/pillars-info/relacje.png";
+import voicePillarImage from "@/assets/images/pillars-info/glos.png";
+
+type Props = {
+  pillar: "relacje" | "glos" | "wplyw";
+};
+
+const PILLAR_DATA = {
```

Comment:
```suggestion
const PILLAR_DATA: Record<PillarName, Pillar> = {
```

Nie jest to błąd, jednak możesz rozważyć dodanie typów o nazwie `Pillar` i `PillarName` i określenia z ich użyciem twardego typowania.

```ts
type PillarName = "relacje" | "glos" | "wplyw";
```

```ts
interface Pillar {
    headerVerb: string;
    headerNoun: string;
    description: JSX.Element
    ...
}
```

Jeśli zdecydujesz się dodać te **opcjonalne** zmiany to zachęcam do użycia ChatuGPT w celu przygotowania właściwego otypowania:)